src/i18n/translations/messages_bg.json

```diff
@@ -33,9 +33,9 @@
   "Server URL": "Адрес на сървър",
   "Why not start the conversation?": "Защо не започнете разговора?",
   "Chat": "Разговор",
-  "Sign in with Google": "Вход с Google",
-  "Sign in with password": "Вход с парола",
-  "Sign in with dev account": "Вход с профил на разработчик",
+  "Log in with Google": "Вход с Google",
```

Comment:
Actually you don't need to changes in all translations files. Just do in `en`. 
(As now it has broken translation.)

Reply:
@jainkuniya Can you please approve it?